.github/workflows/Manager-CI.yaml

```diff
@@ -2,8 +2,8 @@ name: Manager - CI
 on:
```

Comment:
Side note for the future: I think the `Manager-CI` job should be able to run independently without the need of the full data plane. We might review this part 🤔

Reply:
Depends on what should we test there.
For testing just control plane in isolation we can leverage Quarkus tests, done as part of unit testing.